app/services/session_service.rb

```diff
@@ -16,11 +16,10 @@ def authenticate_token(email, token)
   end
 
   def destroy(email)
-    user = User.find_by(email: email)
-    user.update_attribute :auth_token, nil
-    user
-  rescue Mongoid::Errors::DocumentNotFound
-    # find & find_by raises exception if not found
+    if user = User.where(email: email).first
```

Comment:
[Don't use the return value of = (an assignment) in conditional expressions ...](https://github.com/bbatsov/ruby-style-guide#safe-assignment-in-condition)
Anyway you can do:

```ruby
  def destroy(email)
    User.where(email: email).limit(1).detect do |user|
      user.update_attribute :auth_token, nil
    end
  end
```